src/test/java/com/google/api/codegen/testdata/discoveries/go/go_logging.v2beta1.json.baseline

```diff
@@ -22,25 +25,33 @@ import (
 
 func main() {
   ctx := context.Background()
+
+  // Authentication is provided by the `gcloud` tool when running locally
```

Comment:
Yeah, it's kinda strange. ADC is already mentioned in the header, so this should probably be just as simple as

```
// Use Application Default Credentials. See:
// https://....
```

Is this comment the same across languages?